### PR TITLE
feature: add charts 1.9

### PR DESCRIPTION
**Pull request by lut777**
#### What type of this PR
Examples of user facing changes:
- feature

### CODE REVIEW

File: charts/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# KubeOVN-helm
```
Comment (oilbeater) [inline charts/README.md:1]: KubeOVN -> Kube-OVN

**Review by oilbeater — CHANGES_REQUESTED** [review]

File: charts/templates/controller-deploy.yaml [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+      serviceAccountName: ovn
+      hostNetwork: true
+      containers:
+        - name: kube-ovn-controller
+          image: "{{ .Values.image.repository -}}/kube-ovn:{{- .Values.image.version }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+          - /kube-ovn/start-controller.sh
+          args:
+          - --default-cidr=
```
[the remark this answers is not in the document]
Comment (lut777) [reply charts/templates/controller-deploy.yaml]: done

File: charts/README.md (hunk 1 above)
Comment (lut777) [reply charts/README.md]: done

File: charts/values.yaml [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+ingress:
```
Comment (oilbeater) [inline charts/values.yaml:98]: if these values are not used, do not represent them in values.yaml

**Review by oilbeater — CHANGES_REQUESTED** [review]

File: charts/values.yaml (hunk 3 above)
Comment (lut777) [reply charts/values.yaml]: done

**Review by oilbeater — APPROVED** [review]

**Pull request merged**